### ISSUE TITLE
UI: Beginner-Friendly repos page not linked to homepage

### ISSUE THREAD

**Issue by Anikett11** [Sriparno08/Openpedia#263]
## Category
Beginner-Friendly repos page not linked to homepage through openpedia icon.
<img width="1396" alt="Screenshot 2024-01-26 at 10 18 19 PM" src="https://github.com/Sriparno08/Openpedia/assets/104813073/35b45ed2-30f0-4902-8f55-c9bb183cbd65">

- [ ] Documentation
- [ ] Resource Addition
- [x] Codebase
- [ ] User Interface
- [ ] Feature Request

**Comment by Sriparno08** [comment]
You're already assigned to another issue. Once its PR is merged, you will be considered for this issue. If anyone asks to be assigned to this issue before your PR is merged, he/she will be assigned.

**Comment by Anoushka-Aggarwal** [comment]
pls assign it to me @Sriparno08

**Comment by Sriparno08** [comment]
@Anoushka-Aggarwal Please mention 'JWOC' in your comment to be considered for assignment.

**Comment by Anoushka-Aggarwal** [comment]
pls assign it to me @Sriparno08 under JWOC

**Comment by Sriparno08** [comment]
@Anoushka-Aggarwal You're assigned.
